manager/Makefile

```diff
@@ -79,7 +79,14 @@ lint: lint-format lint-code lint-types lint-docs
 lint-format: venv
 	$(VE)/black --check .
 	$(VE)/isort --check-only
-
+	npx js-beautify \
+		--editorconfig \
+		--end-with-newline \
+		--indent-inner-html \
+		--indent-size 2 \
+		--type html \
+		--wrap-line-length 120 \
+		--file **/templates/*.html **/templates/**/*.html
```

Comment:
I feel that this should be an npm script `npm run format` like in our other repos. And then here in this `Makefile` have 

```make
# Format code
format: venv
	$(VE)/black .
	$(VE)/isort --apply
        npm run format
```

Note: under the `format` recipe since this is actually doing writing, not just checking of the format.